apps/hosts/urls.py

```diff
@@ -33,6 +33,11 @@
         views.remove_self_from_challenge_host_team,
         name="remove_self_from_challenge_host_team",
     ),
+    url(
+        r"^challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/challenge_host/(?P<challenge_host_pk>[0-9]+)/delete$",
+        views.delete_challenge_host_from_team,
+        name="delete_challenge_host_from_team",
+    ),
```

Comment:
This would not be required as there is already this URL: https://github.com/Cloud-CV/EvalAI/blob/master/apps/hosts/urls.py#L26-L30